app/controllers/projects_controller.rb

```diff
@@ -150,9 +150,9 @@ def edit
   def update
     @project = Project.find(params[:id])
     #Approve action
-    if params.key?("mediaflux_id")
+    if params.key?("approved")
       @project.metadata_model.update_with_params(params, current_user)
-      @project.approve!(mediaflux_id: params["mediaflux_id"],current_user:)
+      @project.approve!(current_user:)
```

Comment:
The approval method now gets the `mediaflux_id` from Mediaflux rather than we expecting a user to manually enter this value.